saleor/order/calculations.py

```diff
@@ -46,19 +50,33 @@ def _recalculate_order_prices(
                 line_unit = manager.calculate_order_line_unit(
                     order, line, variant, product
                 )
-                line.undiscounted_unit_price = line_unit.undiscounted_price
                 line.unit_price = line_unit.price_with_discounts
 
                 line_total = manager.calculate_order_line_total(
                     order, line, variant, product
                 )
-                line.undiscounted_total_price = line_total.undiscounted_price
                 undiscounted_subtotal += line_total.undiscounted_price
                 line.total_price = line_total.price_with_discounts
 
                 line.tax_rate = manager.get_order_line_tax_rate(
                     order, product, variant, None, line_unit.undiscounted_price
                 )
+                line.undiscounted_unit_price = quantize_price(
+                    calculate_flat_rate_tax(
+                        money=line.undiscounted_base_unit_price,
+                        tax_rate=line.tax_rate * 100,
+                        prices_entered_with_tax=prices_entered_with_tax,
+                    ),
+                    line.currency,
+                )
+                line.undiscounted_total_price = quantize_price(
+                    calculate_flat_rate_tax(
+                        money=line_total.undiscounted_price.net,
```

Comment:
Can you add a comment, that will clarify why you use `.net` here. For someone who is not familiar this can be missleading